generator/views/lookml_utils.py

```diff
@@ -235,7 +239,7 @@ def _is_nested_dimension(dimension: dict):
 
 def render_template(filename, template_folder, **kwargs) -> str:
     """Render a given template using Jinja."""
-    env = Environment(loader=PackageLoader("generator", f"{template_folder}/templates"))
+    env = Environment(loader=FileSystemLoader(PATH / f"{template_folder}/templates"))
```

Comment:
Switching to file system loader here as I was getting `package was not installed in a way that PackageLoader understands` when running this using the multiprocessing pools